docs/preset-env.md

```diff
@@ -229,40 +229,45 @@ This option is useful for "blacklisting" a transform like `@babel/plugin-transfo
 
 `"usage"` | `"entry"` | `false`, defaults to `false`.
 
-This option configures how `@babel/preset-env` handles polyfills.
+This option configures how `@babel/preset-env` imports various polyfills in your generated code.
 
-When either the `usage` or `entry` options are used, `@babel-preset-env` will add direct references to `core-js` modules as bare imports (or requires). This means `core-js` will be resolved relative to the file itself and needs to be accessible.
+Specifically, this option supports the following external polyfill packages:
 
-Since `@babel/polyfill` was deprecated in 7.4.0, we recommend directly adding `core-js` and setting the version via the [`corejs`](#corejs) option.
+- `core-js`: For standard JS built-ins
+- `regenerator`: For generator and `async`/`await` support
 
-```sh
-npm install core-js@3 --save
-
-# or
+Since these polyfills do not come with Babel or `@babel/preset-env` itself, they'll need to be installed as production-level dependencies. This way, when the Babel-generated code imports them, they'll be accessible (in `node_modules`).
 
-npm install core-js@2 --save
```

Comment:
Usage of different `core-js` versions causes many questions and problems for users, so remove the note of how to install different `core-js` versions is not a good idea.

Reply:
That's what I was trying to do by removing the different versions. I found the fact that the documentation had two different versions of `core-js` confusing -- why are there two? Which one should I be using? It made me do unnecessary research into `core-js`, which I don't think a user of `@babel/preset-env` should have to do.

Are there specific use cases for 2 vs 3? If so, I'd be happy to add notes about them 🙂